File: help/pt_pt/exercicios_da_ficha_3/Ficha03_004/Ficha03_004.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha03_004 {

    //Main Function
    static void Main() {
        Console.Write("Teorema de Pitágoras\n" );
        Console.Write("Cateto a \t: " );
        double a = double.Parse(Console.ReadLine());
        Console.Write("Cateto b \t: " );
        double b = double.Parse(Console.ReadLine());
        double c = Math.Pow( a * a + b * b , 0.5 ) ;
        Console.Write("Hipotnusa \t: " + c );
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 14:29

File: help/pt_pt/exercicios_da_ficha_3/Ficha03_004/Ficha03_004.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha03_004 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Teorema de Pitágoras\n" );
        System.out.print("Cateto a \t: " );
        double a = Double.valueOf(keyb.nextLine());
        System.out.print("Cateto b \t: " );
        double b = Double.valueOf(keyb.nextLine());
        double c = Math.pow( a * a + b * b , 0.5 ) ;
        System.out.print("Hipotnusa \t: " + c );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 14:29
